lib/cisco_node_utils/vlan.rb

```diff
@@ -17,10 +17,18 @@
 require_relative 'node_util'
 require_relative 'interface'
 require_relative 'fabricpath_global'
+require_relative 'feature'
 
 # Add some Vlan-specific constants to the Cisco namespace
 module Cisco
   VLAN_NAME_SIZE = 33
+  # rubocop:disable LineLength
+  SAFE_WARNING_MESSAGE = ['Warning: Private-VLAN CLI entered...
```

Comment:
Can we simplify this to be just:

`SAFE_WARNING_MESSAGE` = 'Warning: Private-VLAN CLI entered'`

The additional text does not add much value IMO.